sympy/printing/latex.py

```diff
@@ -2879,12 +2906,23 @@ def latex(expr, full_prec=False, min=None, max=None, fold_frac_powers=False,
     >>> print(latex(log(10), ln_notation=True))
     \ln{\left(10 \right)}
 
-    ``latex()`` also supports the builtin container types list, tuple, and
-    dictionary.
+    ``latex()`` also supports the builtin container types :class:`list`,
+    :class:`tuple`, and :class:`dict`:
 
     >>> print(latex([2/x, y], mode='inline'))
     $\left[ 2 / x, \  y\right]$
 
+    .. versionchanged:: 1.7.0
+        Unsupported types are now embedded as monospaced plaintext:
+
+        >>> print(latex(int))
+        \mathtt{\text{<class 'int'>}}
+        >>> print(latex("plain % text"))
+        \mathtt{\text{plain \% text}}
+
+        See :ref:`printer_method_example` for an example of how to override
+        this behavior for your own types by implementing ``_latex``.
```

Comment:
Does this look good, assuming the link renders correctly?

Reply:
Yes, I think so

Reply:
I’d expect them to be a one line summary like release notes and anything more informative should be immersed in the main docs

Reply:
This is the main docs though, isn't it?

Reply:
This is what I get in the doc build
![Screen Shot 2020-07-23 at 21 56 11](https://user-images.githubusercontent.com/34944973/88288879-561b2d00-cd2f-11ea-82bf-d558f3ac1fae.png)
And I don't see that the indentation is respected in this markup at all

Reply:
I wouldn't expect the indentation to be - I think it gets reflected in the doctree but not the HTML output.

I'll try to fix the hyperlink

Reply:
I'd expect them to have boxed visuals like `.. note`,
Although I see that they are using custom `div` class, I'm not sure that it's just the problem of the outdated CSS themes that makes them render as plain or it's the problem of sphinx.

However, if the sphinx intend them to display in such plain form, it would not be a good idea to contain more than a single sentence summary there.

Reply:
Would you prefer:

```suggestion
    Unsupported types are rendered as monospaced plaintext:

    >>> print(latex(int))
    \mathtt{\text{<class 'int'>}}
    >>> print(latex("plain % text"))
    \mathtt{\text{plain \% text}}

    See :ref:`printer_method_example` for an example of how to override
    this behavior for your own types by implementing ``_latex``.
    
    .. versionchanged:: 1.7.0
        Unsupported types are rendered as monospaced plaintext.
```

Reply:
Yes, although the summary can be duplicate, this is be the best option for now